lib/ami_spec/wait_for_rc.rb

```diff
@@ -0,0 +1,25 @@
+require 'net/ssh'
+
+module AmiSpec
+  class WaitForRC
+    def self.wait(ip_address, user, key)
```

Comment:
Yeah it will. But I can't think of a scenario where this might occur since:

- We already wait for SSH to come up before running this.
- Any failure will cause the command to return

So unless RC never finishes starting (which is possible but unlikely) this will never happen.

In fact I'm going to remove the exception handling around SSH timeouts so it just fails if that fails.